<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorizationDiolog</class>
 <widget class="QDialog" name="AuthorizationDiolog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>69</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="head">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Авторизация</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>26</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="login_input">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>25</height>
            </size>
           </property>
           <property name="plainText">
            <string/>
           </property>
           <property name="overwriteMode">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Логин</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="password_input">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>25</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="login_button">
           <property name="text">
            <string>Войти в кабинет</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>70</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
